### PR TITLE
feat: avro support

### LINKED ISSUE #52: Add format support for avro
Some customers encode their cell values as avro. It would be nice to support avro alongside protobuf.

See b/224602935

### PR DESCRIPTION
**Pull request by sjeandeaux**
The PR contains the Avro support.

Configure a list of avro schema per column to be able to read any version of data.

To read avro data, we must have the schema which with we write the data.

fix #52

### CODE REVIEW

**Comment by sjeandeaux** [comment]
Hi,

How can we enable kokoro?

File: THIRD_PARTY_NOTICES.txt [hunk 1]
```diff
@@ -238,3 +238,31 @@ DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY
...
+   limitations under the License.
+
+----------------------------------------
+NOTICES
+----------------------------------------
+
+goavro
+Copyright 2019 LinkedIn Corp.
+
+Licensed under Apache License 2.0. See LICENSE for terms.
```
Comment (igorbernstein2) [inline THIRD_PARTY_NOTICES.txt:268]: I think we need to add a notice for snappy here

**Review by igorbernstein2 — COMMENTED** [review]
This looks great and thank you for the contribution. I think the only thing thats missing is a notice for the snappy library

File: THIRD_PARTY_NOTICES.txt (hunk 1 above)
Comment (sjeandeaux) [reply THIRD_PARTY_NOTICES.txt:268]: Thanks for the review.

I've add it.

**Comment by sjeandeaux** [comment]
Should we do anything to update kokoro status?

**Comment by sjeandeaux** [comment]
I've just rebased main.

**Comment by sjeandeaux** [comment]
Hi,

Any chance to get a review?

**Comment by sjeandeaux** [comment]
I've just rebased main

**Comment by sjeandeaux** [comment]
> This looks great and thank you for the contribution. I think the only thing thats missing is a notice for the snappy library

It has been fixed

**Comment by sjeandeaux** [comment]
please

**Comment by sjeandeaux** [comment]
Hi @igorbernstein2 ,

Thanks for your feedback. 
Do you know if there is any chance to get a review?

**Comment by sjeandeaux** [comment]
Hi ,

Thanks for your feedback.
Do you know if there is any chance to get a review?

**Comment by sjeandeaux** [comment]
Hi ,

Thanks for your feedback.
Do you know if there is any chance to get a review?

**Comment by sjeandeaux** [comment]
Hi ,

Do you know if there is any chance to get a review?

**Pull request closed without merging**